Automattic/jetpack
# Slideshow Block: Make play/pause icons visible

Fixes the play/pause button mentioned [here](https://github.com/Automattic/wp-calypso/issues/71453#issuecomment-1448395015) in https://github.com/Automattic/wp-calypso/issues/71453

## Proposed changes:

* Change the fill color for the play/pause icons

Before
![before](https://user-images.githubusercontent.com/6586048/223377285-53a26848-e1d5-44a3-9f91-bb27f0e3d70c.png)


After

https://user-images.githubusercontent.com/6586048/223654596-bcafa35f-e594-4f4c-b3ce-63fd6791d33e.mp4




### Other information:


## Jetpack product discussion


None, it's a bug fix

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






1. Pull up a testing site that uses Dara as the theme and has a static homepage. For testing on WP.com run [this command](https://github.com/Automattic/jetpack/pull/29330#issuecomment-1457800976) in sandbox.
2. Add a slideshow to the homepage and let it autoplay
3. Check if the play/pause icons are visible now
